GsaGHTests/3_Components/6_Display/Contour2dResultsTests.cs

```diff
@@ -171,13 +171,6 @@ public void UpdateStressTest() {
       comp.UpdateStress("MPa");
     }
 
-    [Fact]
-    public void UpdateLegendScaleTest() {
-      var comp = new Contour2dResults();
-      ComponentTestHelper.SetInput(comp, GsaResultTests.NodeAndElement2dCombinationResultsMother());
-      comp.UpdateLegendScale();
-    }
```

Comment:
I see no assert here, but why did we remove it? Looks like a clean-up, but was it capturing anything?

Reply:
there is no more updateLegendScale method in contour components. no needed